### PR TITLE
Added four new features concerning doubled values.

### LINKED ISSUE #59: Add feature class: Count/Percentage same value
I had an idea for a class of features. 

1.  percentage of data points that occur at least second time 
2. sum of data points that occur at least second time 
3. percentage of observed values that occur at lest second time 
...

### PR DESCRIPTION
**Pull request by nils-braun**
This closes #59

### CODE REVIEW

**Comment by MaxBenChrist** [comment]
Awesome @nils-braun !

**Comment by nils-braun** [comment]
@MaxBenChrist If you do not want to merge it, you have to give me either access rights or tell me what to implement additionally ;-)

**Comment by MaxBenChrist** [comment]
Damn. I made some comments but forgot to submit them

**Comment by MaxBenChrist** [comment]
I gave you admin rights nils

**Comment by nils-braun** [comment]
No problem and thanks for your comments!

File: tsfresh/feature_extraction/feature_calculators.py [hunk 1]
```diff
@@ -587,6 +587,92 @@ def first_location_of_minimum(x):
...
+    if len(x) == 0:
+        return np.nan
+
+    value_counts = x.value_counts()
+    return 1.0 * value_counts[value_counts > 1].sum() / len(x)
+
+
+@set_property("fctype", "aggregate")
+@not_apply_to_raw_numbers
+def sum_of_doubled_values(x):
```
Comment (nils-braun) [inline tsfresh/feature_extraction/feature_calculators.py]: This is however not the number, but really  the sum of those values (in line 652, you see it is the weigted sum of all those reoccurring datapoints). But sum_of_reocurring_values is good as well.

File: tsfresh/feature_extraction/feature_calculators.py [hunk 2]
```diff
@@ -587,6 +587,92 @@ def first_location_of_minimum(x):
...
+
+
+@set_property("fctype", "aggregate")
+@not_apply_to_raw_numbers
+def percentage_of_reoccurring_values_to_all_values(x):
+    """
+    Returns the ratio of unique value, that are present in the time series
+    more than once.
+
+        # of rows occuring more than once / # of all rows
```
Comment (MaxBenChrist) [inline tsfresh/feature_extraction/feature_calculators.py]: Can you replace the word row with data point? I had a hard time figuring out what you are up to

**Pull request merged**